messages/pt.json

```diff
@@ -5,6 +5,7 @@
   "admin/editor.video.height.title": "Altura",
   "admin/editor.video.autoplay.title": "Reproduzir automaticamente",
   "admin/editor.video.loop.title": "Reproduzir em loop",
+  "admin/editor.video.playsInline.title": "Reproduzir in-line (em dispositivos móveis)",
```

Comment:
`in-line` in portuguese, intersting

Reply:
yup, I was surprised too! But hey, `em linha` sounds even weirder 😆